jaxb-jakarta/src/main/java/feign/jaxb/JAXBContextFactory.java

```diff
@@ -51,6 +78,10 @@ public Unmarshaller createUnmarshaller(Class<?> clazz) throws JAXBException {
   public Marshaller createMarshaller(Class<?> clazz) throws JAXBException {
     Marshaller marshaller = getContext(clazz).createMarshaller();
     setMarshallerProperties(marshaller);
+    if (marshallerEventHandler != null) {
```

Comment:
So, this is piling more and more functionality/code into feign to allow fine tuning...  wonder if it would be possible to turn this into an interface that allows user to do whatever they to `JAXBContext` and pass it to feign in a more transparent way..